packages/e2e-tests/src/specs/dashboard/dashboard.js

```diff
@@ -43,8 +46,12 @@ describe('Stories Dashboard', () => {
   });
 
   it('should choose sort option for display', async () => {
+    // dropdown needs a story for filtering
```

Comment:
Noticed the dashboard test was failing ...

The dropdown didn't have a story to work with.

Reply:
I cherry picked that commit for this PR

https://github.com/GoogleForCreators/web-stories-wp/pull/11227